examples/06-kitchen-sink/src/block/block.js

```diff
@@ -0,0 +1,482 @@
+/**
+ * BLOCK: kitchen-sink
+ *
+ * Registering a basic block with Gutenberg.
+ * Simple block, renders and saves the same content without any interactivity.
+ */
+
+//  Import CSS.
+import "./style.scss";
+import "./editor.scss";
+
+const { __ } = wp.i18n; // Import __() from wp.i18n
+const { registerBlockType } = wp.blocks; // Import registerBlockType() from wp.blocks
+const { Fragment } = wp.element; // Import Fragment from wp.elements
+
+// Register editor components
```

Comment:
Document addition of each new module. Don't be afraid of commenting about everything. This example is meant to teach people.